### PR TITLE
test: Fix review goods test and also update conf test to read from .env

### PR DESCRIPTION
**Pull request by phlptt**

### CODE REVIEW

File: ui_automation_tests/features/review_goods.feature [hunk 1]
```diff
@@ -11,5 +11,5 @@ Feature: I want to review, amend where required and confirm the goods ratings an
     When I go to application previously created
     And I select goods and click review
     And I click on add report summary
-    And  I respond "yes", "ML4b1", "0", "Because the good is controlled" and click continue
+    And  I respond "yes", "ML4b1", "1", "Because the good is controlled" and click continue
```
Comment (jfaracik) [inline ui_automation_tests/features/review_goods.feature]: Two spaces after and

File: ui_automation_tests/conftest.py [hunk 2]
```diff
@@ -34,8 +34,8 @@ def pytest_addoption(parser):
     parser.addoption("--sso_sign_in_url", action="store", default="https://sso.trade.uat.uktrade.io/login/", help="url")
 
     if env.lower() == 'local':
-        parser.addoption("--internal_url", action="store", default="http://localhost:8200", help="url")
-        parser.addoption("--lite_api_url", action="store", default="http://localhost:8100", help="url")
+        parser.addoption("--internal_url", action="store", default="http://localhost:" + str(os.environ.get('PORT')), help="url")
+        parser.addoption("--lite_api_url", action="store", default=str(os.environ.get('LITE_API_URL')), help="url")
```
Comment (BenAndrew-Kainos) [inline ui_automation_tests/conftest.py]: I thought we did this before 😕

Comment (phlptt) [reply ui_automation_tests/conftest.py]: hey yea it was done on exporter not internal

Comment (BenAndrew-Kainos) [reply ui_automation_tests/conftest.py]: ah ok

**Pull request merged**